docs/sources/dashboards/time-range-controls.md

```diff
@@ -0,0 +1,116 @@
++++
+title = "Time range controls"
+keywords = ["grafana", "dashboard", "documentation", "time range"]
+type = "docs"
+[menu.docs]
+aliases = ["/docs/grafana/latest/reference/timerange/"]
+name = "Time range controls"
+parent = "dashboards"
+weight = 7
++++
+
+# Time range controls
+
+Grafana provides several ways to manage the time ranges of the data being visualized, both at the dashboard level and the panel level. 
+
+This page describes supported time units and relative ranges, the common time controls, dashboard-wide time settings, and panel-specific time settings.
+
+## Time units and relative ranges
+
+The following time units are supported: `s (seconds)`, `m (minutes)`, `h (hours)`, `d (days)`, `w (weeks)`, `M (months)`, and `y (years)`. 
+
+The minus operator allows you to step back in time, relative to now. If you wish to display the full period of the unit (day, week, month, etc...), append `/$unit` to the end.
+
```

Comment:
I think a paragraph which explains that the plus operator can be used to step forward in time relative to now should also be added. Maybe not used by everyone, but there are users who uses this feature to look at predicted data in the future for example.

Reply:
Added